src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,10 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) =>
+  //    if (boolean === true) {
+  //        return 'Yes';
+  //    } else {
+  //        return 'No';
+  //    }
+  boolean ? "Yes" : "No";
+// refactored code using ternary operator ?
```

Comment:
You should remove all the comments here, you have a working solution with the ternary so thats all you need to include here! 
Once you do, you put this all on one line, dont even need to use the return keyword as its implied when putting a function on a single line like this